ruby/lib/minitest/queue.rb

```diff
@@ -268,8 +268,10 @@ def run_from_queue(reporter, *)
       exit!(41)
     rescue => error
       reopen_previous_step
+      queue.report_worker_error(error)
       puts red("This worker exited because of an uncaught application error:")
       puts red("#{error.class}: #{error.message}")
+      puts red("#{error.backtrace}")
```

Comment:
That will call `Array#to_s`, that's not what you want:

```
>> puts "#{["foo", "bar"]}"
["foo", "bar"]
```

```suggestion
      error.backtrace.each do |frame|
        puts red(frame)
      end
```